src/org/omegat/util/PluginInstaller.java

```diff
@@ -109,17 +107,17 @@ public static boolean install(final File pluginFile) {
         }
 
         // confirm installation
-        if (JOptionPane.YES_OPTION == JOptionPane.showConfirmDialog(Core.getMainWindow().getApplicationFrame(),
-                    message,
-                    OStrings.getString("PREFS_PLUGINS_TITLE_CONFIRM_INSTALLATION"),
-                    JOptionPane.OK_CANCEL_OPTION, JOptionPane.ERROR_MESSAGE)) {
+        if (JOptionPane.YES_OPTION == JOptionPane.showConfirmDialog(
+                Core.getMainWindow().getApplicationFrame(), message,
+                OStrings.getString("PREFS_PLUGINS_TITLE_CONFIRM_INSTALLATION"), JOptionPane.OK_CANCEL_OPTION,
+                JOptionPane.ERROR_MESSAGE)) {
```

Comment:
```suggestion
                JOptionPane.WARNING_MESSAGE)) {
```

The message is warning not error. Also we should update message

"PREFS_PLUGINS_CONFIRM_UPGRADE"

and 

"PREFS_PLUGINS_CONFIRM_INSTALL"

Reply:
Thank you. @Kazephil could you take a look at that?

Reply:
Sure, but it might take a few days. (It looks like I won't get a whole lot of time at home this weekend.)

Reply:
also "PREFS_PLUGINS_TITLE_CONFIRM_INSTALLATION"

Reply:
also "PREFS_PLUGINS_INSTALLATION_FAILED" that dialog will be shown when installation error occurred.